guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -1,32 +1,62 @@
 [id="Using_Container_Registries_{context}"]
 = Using container registries
 
-Podman and Docker can be used to fetch content from container registries.
+You can use Podman and Docker to fetch content from container registries and push containers to the {Project} container registry.
+The {Project} registry follows the Open Containers Initiative (OCI) specification, so you can push content to {Project} by using the same methods that apply to other registries.
+For more information about OCI, see link:https://opencontainers.org/[Open Container Initiative Distribution Specification].
+
+.Prerequisites
+* To push content to an organization or product, a user must have permissions to create repositories.
+The push permission is `edit_products`.
+* To pull content, a user must have permissions as long as *unauthenticated pull* is set to off by default in the lifecycle environment.
+The pull permissions are `view_lifecycle_environments`, `view_products`, `view_content_views`.
 
 ifndef::orcharhino[]
 .Container registries on {SmartProxies}
-
 On {SmartProxies} with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] {SmartProxy} plugin acts as the container registry.
 It caches authentication information from Katello and proxies incoming requests to Pulp.
 The Container Gateway is available by default on {SmartProxies} with content.
 endif::[]
 
-.Procedure
+.Considerations for pushing content to the {Project} container registry
+* You can only push content to the {ProjectServer} itself.
+If you need pushed content on {SmartProxyServers} as well, use {SmartProxy} syncing.
+* The pushed container name must contain only lowercase characters.
```

Comment:
Consistency. We talk about container name here but we use container hash below. Does it mean that the user can use either?